lib/python/ensembl/brc4/runnable/integrity.py

```diff
@@ -119,18 +137,28 @@ def run(self):
 
             # Check gff3
             if gff:
+                #Check fasta_pep.fa integrity
+                #The sequence length and id retrieved from the fasta_pep file and compared to the translated CDS id and length in the gff
+                # We don't compare the peptide lengths because of seqedits
                 if pep:
-                    # We don't compare the peptide lengths because of seqedits
                     tr_errors = self.check_lengths(pep, gff["translations"], "Fasta translations vs gff", special_diff = True)
                     if len(tr_errors) > 0:
+                        #The psuedo CDSs are included in this check
+                        #Psuedo CDSs are not translated,if the pseudo translation ids are not ignored in the gff it will give an error
```

Comment:
```suggestion
                        # Pseudo CDSs are not translated, if the pseudo translation ids are not ignored in the gff it will give an error
```